audio/mixer_intern.h

```diff
@@ -82,7 +82,7 @@ class MixerImpl : public Mixer {
 
 public:
 
-	MixerImpl(uint sampleRate, uint outBufSize = -1);
+	MixerImpl(uint sampleRate, uint outBufSize = 0);
```

Comment:
can you squash this change to earlier commit which introduce -1

Reply:
Sure, I'll look into it and do it

Reply:
There you go, should be much more cleaner than before